lib/services/TypeService.js

```diff
@@ -0,0 +1,196 @@
+let exportObj = {};
```

Comment:
I think we may want to consider moving the whole project to TypeScript before it gets too large instead of using something like this.